ql/src/test/results/clientpositive/llap/subquery_notin.q.out

```diff
@@ -5300,20 +5300,27 @@ STAGE PLANS:
                         Statistics: Num rows: 16 Data size: 1728 Basic stats: COMPLETE Column stats: COMPLETE
                         value expressions: _col1 (type: bigint), _col2 (type: bigint)
                   Filter Operator
-                    predicate: (p_brand is not null and p_type is not null and UDFToDouble(p_type) is not null) (type: boolean)
+                    predicate: (p_brand is not null and p_type is not null) (type: boolean)
                     Statistics: Num rows: 26 Data size: 5096 Basic stats: COMPLETE Column stats: COMPLETE
-                    Group By Operator
-                      keys: p_brand (type: string), p_type (type: string)
-                      minReductionHashAggr: 0.4
-                      mode: hash
+                    Select Operator
+                      expressions: p_type (type: string), p_brand (type: string)
                       outputColumnNames: _col0, _col1
-                      Statistics: Num rows: 24 Data size: 4704 Basic stats: COMPLETE Column stats: COMPLETE
-                      Reduce Output Operator
-                        key expressions: _col0 (type: string), _col1 (type: string)
-                        null sort order: zz
-                        sort order: ++
-                        Map-reduce partition columns: _col0 (type: string), _col1 (type: string)
-                        Statistics: Num rows: 24 Data size: 4704 Basic stats: COMPLETE Column stats: COMPLETE
+                      Statistics: Num rows: 26 Data size: 5096 Basic stats: COMPLETE Column stats: COMPLETE
+                      Filter Operator
+                        predicate: UDFToDouble(_col0) is not null (type: boolean)
```

Comment:
Rather minor but it seems that now some filter operators cannot be merged together.